src/main/java/frc/robot/subsystems/DefaultTeleopSub.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class DefaultTeleopSub extends SubsystemBase{
+    
+
+    public Swerve s_Swerve;
+    public Limelight s_Limelight;
+    private static DefaultTeleopSub autoAlignSwerve;
+
+    public static DefaultTeleopSub getInstance() {
+        if (autoAlignSwerve == null) {
+            autoAlignSwerve = new DefaultTeleopSub();
+        }
+        return autoAlignSwerve;
+    }
+
+    public DefaultTeleopSub() {
```

Comment:
Should be private because we don't want the constructor to be called else where.